test/config/test_config.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+import os
+import json
+from datetime import datetime
+from src.config import config
+from src.config import BoundsResult
+
+
+@pytest.fixture
+def empty_config():
+    return config
+
+@pytest.fixture
+def loaded_config(tmp_path):
+    mock_config = {
+        "email": "test@test.com",
+        "pipeline_user": "sw23",
+        "pipeline_passwd": "fake",
+        "pipeline_hostname": "localhost",
+        "pipeline_db_name": "aperature",
+        "columns": { "vehicle_number": { "max": "NA", "min": 0 }, "maximum_speed" : {"max" : 150, "min" : 0}, "service_date" : {"max" : "NA", "min" : "1990-01-01"} }
+    }
+
+    p = tmp_path / "test_config.json"
+    p.write_text(json.dumps(mock_config))
+    config.load(p)
+    return config
+    
+def test_get_set_config(empty_config):
+    config.set_value("test", 5)
+    assert config.get_value("test") == 5
+
+def test_ingest_env(monkeypatch, empty_config):
+    monkeypatch.setenv("PIPELINE_USER", "test_user")
+    monkeypatch.setenv("PIPELINE_PASSWD", "test_pass")
+    empty_config.ingest_env()
+
+    assert empty_config.get_value("pipeline_user") == "test_user"
+    assert empty_config.get_value("pipeline_passwd") == "test_pass"
+
+def test_load_config(tmp_path):
+    mock_config = {
+        "email": "test@test.com",
+        "pipeline_user": "sw23",
+        "pipeline_passwd": "fake",
+        "pipeline_hostname": "localhost",
+        "pipeline_db_name": "aperature",
+        "columns": { "vehicle_number": { "max": "NA", "min": 0 } }
+    }
```

Comment:
Can't you use the same mock_config as the above fixture? Just so that you don't have duplicate data just sitting around.

Reply:
I'll fix this